### PR TITLE
Contributor's Guide: Create "Using Github Issues"

### PR DESCRIPTION
**Pull request by BD103**
Part of #1146. This covers both a guide on when to create issues, as well as a more general-purpose triage reference.

### CODE REVIEW

**Review by alice-i-cecile — COMMENTED** [review]
Good content! I think this page is in a bit of a weird place: it's talking to two different audiences. The first is "I'm a random user who wants to file an issue" and the second is "I want to do or understand triage". The intro and "When to File an Issue" feel like they should be lumped into a "Filing Issues" doc, while the rest belongs in a "Bevy's Triage Process" doc.

In the second doc, we should also make it very clear that we hand out triage rights like candy, and to just ask if you have any interest.

**Comment by BD103** [comment]
I've split up the document into a page on Github Issues and a page on triage. Tell me what you think! Does "Using Github Issues" now need more content, or does it serve its purpose?

**Review by alice-i-cecile — COMMENTED** [review]
I like the split! I've left advice for you on the Triage section in a comment :)

We should have a section in "Using Github Issues" on when to open a Discussion instead. That should flesh it out nicely.

File: content/contributing/reference/triage.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
++++
+title = "Triage"
+insert_anchor_links = "right"
+[extra]
+weight = 1
++++
+
+<!-- TODO: Introduction -->
```
[the remark this answers is not in the document]
Comment (BD103) [reply content/contributing/reference/triage.md]: Thank you! Added in 1c99edb03b8d9219464c5d66f55a56931c8c1643.

**Review by alice-i-cecile — APPROVED** [review]

File: content/contributing/reference/triage.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+Bevy's scope and user base means that it gets a large volume of issues and pull requests. This work all needs to be organized: making it easier to search through to find related issues, to figure out where you can help, and to quickly communicate the status of a task at a glance. The continual process of organizing work is called **triage**.
+
+## Classifying issues and PRs
+
+Labels are our primary tool for organizing work. You can find a complete list with descriptions [here](https://github.com/bevyengine/bevy/labels). These are a few of the most common, organized by category:
+
+- **A**: Area (e.g. `A-Animation`, `A-ECS`, `A-Rendering`, ...).
+- **C**: Category. The most common include:
+    - `C-Bug`: unexpected or incorrect behavior.
+    - `C-Enhancement`: a new feature.
```
Comment (mockersf) [inline content/contributing/reference/triage.md:17]: not just a new feature, it can be an enhancement of an existing one

Comment (alice-i-cecile) [reply content/contributing/reference/triage.md:17]:
```suggestion
    - `C-Enhancement`: a new feature or meaningful extension of an existing one.
```

**Comment by BD103** [comment]
I'm writing one quick section on Github Discussions, then this is good to go!

File: content/contributing/getting-started/using-github-issues.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Bevy tracks bugs and feature requests using [Github Issues], which is an issue tracker and discussion space. You can find Bevy's issue page [here](https://github.com/bevyengine/bevy/issues), where you can navigate, filter, and view individual issues.
+
+[Github Issues]: https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues
+
+## When to file an issue
+
+Issues are great for formally tracking the progress of bugs and features. Here are some scenarios where you may choose to use issues:
+
+- You want to receive feedback and ideas before writing code.
```
Comment (ramirezmike) [inline content/contributing/getting-started/using-github-issues.md:16]: should we clarify that "writing code" here means specifically for contributing to Bevy and not just code using Bevy?

File: content/contributing/reference/triage.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
++++
+title = "Triage"
+insert_anchor_links = "right"
+[extra]
+weight = 1
++++
+
+Bevy's scope and user base means that it gets a large volume of issues and pull requests. This work all needs to be organized: making it easier to search through to find related issues, to figure out where you can help, and to quickly communicate the status of a task at a glance. The continual process of organizing work is called **triage**.
```
Comment (ramirezmike) [inline content/contributing/reference/triage.md:8]: this is a minor nitpick to simplify this sentence a bit and I'm cool with it either way (I realize this was originally suggested by Alice)

>This work all needs to be organized to make it easier to find related issues, figure out where you can help, and to quickly communicate the status of a task at a glance.

Comment (alice-i-cecile) [reply content/contributing/reference/triage.md:8]: Yep, I'm on board with that.

File: content/contributing/getting-started/using-github-issues.md (hunk 3 above)
Comment (alice-i-cecile) [reply content/contributing/getting-started/using-github-issues.md:16]:
```suggestion
- You want to receive feedback and ideas before writing code for Bevy.
```

**Review by ramirezmike — APPROVED** [review]

**Pull request merged**